recipe/meta.yaml

```diff
@@ -1,37 +1,31 @@
 {% set name = "streamlit-extras" %}
-{% set version = "0.2.7" %}
+{% set version = "0.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/streamlit_extras-{{ version }}.tar.gz
-  sha256: 52a8cb3a4394735ee873c92c95fabf56d84cc2400ed0617bbc8b8259c0560276
-  patches:
-    # entrypoints is a package in maintenance mode that seems to be implicitly used by streamlit-extras. It seems to be 
-    # not really needed. Rather than adding entrypoints to the runtime dependencies, it was decided to remove it from the implementation.
-    - patches/0001-remove-the-usage-of-entrypoints.patch
+  sha256: ac67645ab84accb5ae4de8ef7ca5dd5fc69965d934b9373f813770720814204d
 
 build:
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
   # s390x is missing streamlit
   skip: true  # [py<38 or s390x]
```

Comment:
need to  replicate `<3.9.7,>3.9.7,<4.0` in skip to match this in their pyproject.toml

https://github.com/arnaudmiribel/streamlit-extras/blob/28c9d995c9b4a3ca04484fa4b3bb9e34855149f6/pyproject.toml#L20C1-L20C87

Reply:
I added a python pinning of `!=3.9.7` for python under `host` and `run`